.github/workflows/python-app.yml

```diff
@@ -36,7 +37,7 @@ jobs:
           POSTGRES_PASSWORD: postgres
           PGPASSWORD: postgres
         ports:
-          - "5432:5432"
+          - "5433:5432"
```

Comment:
Just curious why this is necessary.

Reply:
This change is purely for convenience. I usually run a local PostgreSQL instance on port 5432 for development. When testing the GitHub Action locally using 'act', I need to shut down my local PostgreSQL instance to avoid port conflicts. By changing the port, I can avoid this issue and keep both services running simultaneously :)

Reply:
Sounds good. Please leave a comment explaining this.

Reply:
Done.